pkg/cmd/cli/restore/logs.go

```diff
@@ -53,3 +54,33 @@ func NewLogsCommand(f client.Factory) *cobra.Command {
 
 	return c
 }
+
+type LogsOptions struct {
+	RestoreName string
+	client      arkclient.Interface
+}
+
+func NewLogsOptions() *LogsOptions {
```

Comment:
I know we've been remiss up till now, but would you mind doing godoc for anything that's exported?

Reply:
ack, I've added comments for the exported fields added in this PR